templates/taxi/driver_list.html

```diff
@@ -1,7 +1,9 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1>Driver List <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
```

Comment:
Ensure that the URL for 'taxi:driver-create' is correctly defined in the `urls.py` file. If the URL pattern is incorrect, this link will not work as expected.